### ISSUE TITLE
Meta-q intermittently replaces two linefeeds with a single space

### ISSUE THREAD

**Issue by peoppenheimer** [jessealama/ce-el#17]
It's hard to provide an example, because it seems to be non-deterministic.  Sometimes, meta-q will reformat a block element in such a way as to violate the SEP convention of having a blank line between the start tag and the contents.  So, sometimes:

```
<p>

blah blah blah
quux quux quux</p>
```

will be changed by meta-q not to the desired:

```
<p>

blah blah blah quux quux quux</p>
```

but rather to:

```
<p> blah blah blah quux quux quux</p>
```

**Comment by jessealama** [comment]
At the moment, ce-el does not (re)bind C-q, so what you're seeing comes from the (busted, ancient) built-in HTML mode for Emacs.

It is easy enough to make a function that enforces the SEP convention for the whole buffer.  This would be a whole-buffer operation, not a "paragraph"-based operation, which is what M-q is supposed to be.  The idea would be that you could edit "paragraphs" however you wish, and then, when you're ready, apply this (not-yet-created) function to fix all paragraphs (not just the current paragraph).

**Comment by peoppenheimer** [comment]
Yes, a global fix-up to enforce the SEP convention sounds like the right thing.  Then paragraphs could be wrapped, unwrapped, or whatever is best during editing operations, and tidied up at the very end, like a carpenter finishing bookcases with molding.
